docs/caa.md

```diff
@@ -0,0 +1,107 @@
+---
+layout: page
+title: Certificate Authority Authorization (CAA)
+permalink: /docs/caa/
+top_graphic: 1
+date: 2017-07-27T00:00
+---
+
+Last updated: {{ page.date | date: "%B %d, %Y" }} \| [See all Documentation](/docs/)
+
+CAA is a type of DNS record that allows site owners to specify which Certificate
+Authorities (CAs) are allowed to issue certificates containing their domain names. It
+was standardized in 2013 by [RFC 6844](https://tools.ietf.org/html/rfc6844) to
+allow a CA "reduce the risk of unintended certificate mis-issue." By default,
+every public CA is allowed to issue certificates for any domain name in the
+public DNS, provided they validate control of that domain name. That means that
+if there's a bug in any one of the many public CAs' validation processes, every
+domain name is potentially affected. CAA provides a way for domain holders to
+reduce that risk.
+
+# Using CAA
+
+If you don't care about CAA, you generally don't have to do anything (but see
+CAA errors below). If you would like to use CAA to restrict which Certificate
+Authorities are allowed to issue certificates for your domain, you will need to
+use a DNS provider that supports setting CAA records. Check [SSLMate's CAA
+page](https://sslmate.com/labs/caa/#support) for a list of such providers. If your
+provider is listed, you can use
+[SSLMate's CAA Record Generator](https://sslmate.com/labs/caa/) to generate a
+set of CAA records listing the CAs that you would like to allow.
+
+## Where to put the record
+
+You can set CAA records on your main domain, or at any depth of subdomain.
+For instance, if you had `www.community.example.com`, you could set CAA records
+for the full name, or for `community.example.com`, or for `example.com`. CAs
+will check each version, from left to right, and stop as soon as they see any
+CAA record. So for instance, a CAA record at `community.example.com` would take
```

Comment:
Huh! I guess that means we told someone on the forums from a university to give up unnecessarily on getting certificates issued for his subdomain.